config/profile/rhdh/default-config/db-statefulset.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: backstage-psql-cr1 # placeholder for 'backstage-psql-<cr-name>'
 spec:
   podManagementPolicy: OrderedReady
-  replicas: 1
+  # replicas: 1 # Intentionally omitted to allow HPA or custom scaling control.
```

Comment:
[config/profile/backstage.io/default-config/db-statefulset.yaml](https://github.com/redhat-developer/rhdh-operator/blob/main/config/profile/backstage.io/default-config/db-statefulset.yaml) should also be updated.